programs/sbf/Cargo.lock

```diff
@@ -630,7 +630,7 @@ source = "registry+https://github.com/rust-lang/crates.io-index"
 checksum = "4114279215a005bc675e386011e594e1d9b800918cea18fcadadcce864a2046b"
 dependencies = [
  "borsh-derive 0.10.3",
- "hashbrown 0.12.3",
+ "hashbrown 0.11.2",
```

Comment:
Huh, kind of weird that this changed. It looked like it alternated the other way on last version bump 🤔 :

https://github.com/solana-labs/solana/pull/32419/files#diff-13ee4b2252c9e516a0547f2891aa2105c3ca71c6d7a1e682c69be97998dfc87eL662-R662

FYI @willhickey

Reply:
Yeah I haven't been able to figure out what caused this. Potentially related to borsch version changes but I haven't nailed it down.

I'll keep an eye on it in case it causes problems